Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1520,6 +1522,11 @@ - (NSString *)cacheKeyForURL:(NSURL *)url processorKey:(NSString *)processorKey
 
 - (void)objectForKey:(NSString *)key options:(PINRemoteImageManagerDownloadOptions)options completion:(void (^)(BOOL found, BOOL valid, PINImage *image, id alternativeRepresentation))completion
 {
+    if ((options & PINRemoteImageManagerDownloadOptionsIgnoreCache) != 0) {
+        completion(NO, NO, nil, nil);
```

Comment:
👍  I'll fix that as well, I matched that values here for when the object isn't in the cache - https://github.com/pinterest/PINRemoteImage/blob/master/Pod/Classes/PINRemoteImageManager.m#L1545.

It looks like we'd have the same problem here, right? https://github.com/pinterest/PINRemoteImage/blob/master/Pod/Classes/PINRemoteImageManager.m#L1100.